.eslintrc.json

```diff
@@ -74,6 +74,7 @@
     "react/prop-types": "off",
     "react/require-default-props": "off",
     "import/newline-after-import": "off",
+    "flow-header/flow-header": 2,
```

Comment:
Did you find out what the 2 means?

Reply:
Yeah, it means "error". I replaced it by "error", since this more clear.